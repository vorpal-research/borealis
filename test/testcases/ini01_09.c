#include <stdio.h>

extern int arr[10];

int getArrayElem(const int* arr, int index)
{
   return arr[index];
}

int main(void)
{
   int index;
   //scanf("%d",&index);
   if (index >= 0)
      return 0;
   // Здесь должен быть только BUF-02
   return getArrayElem(arr, index);
}