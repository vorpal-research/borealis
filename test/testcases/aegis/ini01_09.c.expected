[
    {
        "location": {
            "loc": {
                "col": 4,
                "line": 6
            },
            "filename": "test/testcases/aegis/ini01_09.c"
        },
        "type": "BUF-01"
    },
    {
        "location": {
            "loc": {
                "col": 4,
                "line": 6
            },
            "filename": "test/testcases/aegis/ini01_09.c"
        },
        "type": "INI-03"
    }
]